examples/demo.css

```diff
@@ -347,7 +347,18 @@ a.name-header{
  left: 10%;
  top: 30px;
 }
-#update-prompt-modal.show {
+
+#notify-box {
```

Comment:
Wow cool, it looks like you created a custom alert box style, was there a reason why the bootstrap styles didn't work for you? Thanks!!!

Reply:
There is no special reason for this .but the thing which took me to create a custom one is because toast is available in bootstrap version v4 and currently we have v3 so either we have to use cdn links or update it which i feel not to do so just for a notification prompt.
But it worth having this custom prompt bar.

Reply:
@jywarren sir is it ok😊

Reply:
OK, that makes sense! And, it looks nice, good work!

![image](https://user-images.githubusercontent.com/24359/108531258-7d75b080-72a4-11eb-8ea7-91886ba2829a.png)


Let's do this: let's use a classname instead of an ID so it can be more easily reused, and let's also use the same classname as the Bootstrap 4 code -- that way if we upgrade in the future, it'll be a relatively easy change where we remove our custom style and switch over. How does that sound? 

Thank you for your help on this!!!

Reply:
That sounds perfect!! . And sir you are right we should make this code a little bit more general in nature for further reusability. i will change it now 😊 .

Reply:
BTW i was just planning to provide you a visual for the changes i have made so that you can judge it better ,but before that you send this pick isn't it funny